sdk/cosmosdb/Azure.ResourceManager.CosmosDB/src/autorest.md

```diff
@@ -506,6 +519,30 @@ directive:
 - rename-model:
     from: RestorableMongodbDatabaseGetResult
     to: RestorableMongoDBDatabase
+- rename-model:
+    from: RestorableGremlinDatabaseGetResult
+    to: RestorableGremlinDatabase
+- rename-model:
+    from: RestorableGremlinGraphGetResult
+    to: RestorableGremlinGraph
+- rename-model:
+    from: RestorableTableGetResult
+    to: RestorableTable
+- rename-model:
+    from: ClientEncryptionKeyGetPropertiesResource
+    to: CosmosDBSqlClientEncryptionKeyProperties
+- rename-model:
+    from: ClientEncryptionKeyResource
+    to: CosmosDBSqlClientEncryptionKeyResource
+- rename-model:
+    from: ClientEncryptionPolicy
+    to: CosmosDBClientEncryptionPolicy
+- rename-model:
+    from: ClientEncryptionIncludedPath
+    to: CosmosDBClientEncryptionIncludedPath
+- rename-model:
+    from: ClientEncryptionKeyGetResultData
```

Comment:
this is not working. Please remove the Data suffix here and it might work

Reply:
Fixed